### ISSUE TITLE
Score System : Traps doesn't count well

### ISSUE THREAD

**Issue by Anfournier** [VroumStudio/REKT#41]
At the end of the race, when the score is display, it seems it count only the traps still ON track and not ALL the traps we use during the race.
It's a problem if we use the mine cause they are destroy after use and it doesn't count in the final score.

**Comment by tbeaugelin** [comment]
Yes, the current method display only the remaining traps at the end of the race. Not all the traps that have been placed during the race. It should be easy to fix with an int being incremented each time a trap is placed.
